docs/components/Countdown/Web.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { Countdown } from "@jobber/components/Countdown";
+import { Button } from "@jobber/components/Button";
+import { Emphasis } from "@jobber/components/Emphasis";
+import { Content } from "@jobber/components/Content";
+import { Heading } from "@jobber/components/Heading";
+import { Text } from "@jobber/components/Text";
+
+export default {
+  title: "Components/Utilities/Countdown/Web",
+  component: Countdown,
+  parameters: {
+    viewMode: "story",
+    previewTabs: { code: { hidden: false } },
+  },
+} as ComponentMeta<typeof Countdown>;
+
+const BasicTemplate: ComponentStory<typeof Countdown> = args => (
+  <Countdown {...args} />
+);
+
+export const Basic = BasicTemplate.bind({});
+Basic.args = {
+  date: new Date(new Date().getTime() + 25 * 3600 * 1000),
+  granularity: "dhms",
+  showUnits: true,
+};
+
+const OnCompleteTemplate: ComponentStory<typeof Countdown> = args => {
+  const [start, setStart] = useState(false);
+  const inTenSeconds = new Date(new Date().getTime() + 10 * 1000);
+  return (
+    <div>
+      <Button label={"Start Timer"} onClick={() => setStart(true)} />
+      {start && (
+        <Emphasis variation={"highlight"}>
```

Comment:
Not a big deal but I don't think we need emphasis here. We have the Styling example that uses it below

Reply:
I changed it to emphasis because the old file was like:
`<Heading variation={"highlight"}>`

It's not even a thing!! 🤣